tasks/users.yml

```diff
@@ -3,6 +3,7 @@
   postgresql_user:
     name: "{{ item.name }}"
     password: "{{ item.password | default(omit) }}"
+    encrypted: yes
```

Comment:
I think if anything, this should be set like:

```
encrypted: "{{ item.encrypted | default(omit) }}"
```

Reply:
See: https://github.com/geerlingguy/ansible-role-postgresql/pull/33 — please remove this change from the PR and I'll be happy to merge.